src/Microsoft.ComponentDetection.Detectors/pip/SimplePipComponentDetector.cs

```diff
@@ -0,0 +1,133 @@
+namespace Microsoft.ComponentDetection.Detectors.Pip;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Microsoft.ComponentDetection.Contracts;
+using Microsoft.ComponentDetection.Contracts.Internal;
+using Microsoft.ComponentDetection.Contracts.TypedComponent;
+using Microsoft.Extensions.Logging;
+
+public class SimplePipComponentDetector : FileComponentDetector, IExperimentalDetector
```

Comment:
Can we mark this ass DefaultOff for now, merge this, iterate over performance concerns, then we can:

1. Leave as DefaultOff and let some customers test this new detector for us
2. Run as experimental

Reply:
updated it